common/src/test/java/com/google/auto/common/SuperficialValidationTest.java

```diff
@@ -274,7 +274,7 @@ public void invalidAnnotationValue() {
   }
   */
 
-  private abstract static class AssertingProcessor extends AbstractProcessor {
```

Comment:
Google style and the [JLS](http://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.1.1) both say that this is already the correct order. Where is the other one coming from?